hack/jenkins/e2e.sh

```diff
@@ -602,6 +602,7 @@ case ${JOB_NAME} in
     : ${E2E_UP:="true"}
     : ${E2E_TEST:="false"}
     : ${E2E_DOWN:="false"}
+    : ${ENABLE_DEPLOYMENTS:=true}
```

Comment:
I think this is no longer neccessary now that #15024 is merged. It should be enabled now.